testing/py-packaging/APKBUILD

```diff
@@ -1,22 +1,28 @@
 # Maintainer: Breno Leitao <breno.leitao@gmail.com>
 pkgname=py-packaging
+_pkgname=packaging
 pkgver=16.8
-pkgrel=0
+pkgrel=1
 pkgdesc="Core utilities for Python packages"
 url="https://pypi.python.org/pypi/packaging"
-arch="all"
+arch="noarch"
 license="APACHE or BSD"
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 makedepends="python3-dev python2-dev py-setuptools"
-source="https://pypi.python.org/packages/c6/70/bb32913de251017e266c5114d0a645f262fb10ebc9bf6de894966d124e35/packaging-$pkgver.tar.gz"
-builddir="$srcdir/packaging-$pkgver"
+source="https://pypi.python.org/packages/c6/70/bb32913de251017e266c5114d0a645f262fb10ebc9bf6de894966d124e35/$_pkgname-$pkgver.tar.gz"
```

Comment:
You could use this:

https://files.pythonhosted.org/packages/source/p/$_pkgname/$_pkgname-$pkgver.tar.gz